### DISCUSSION TITLE
Mount Btrfs

### DISCUSSION THREAD

**Discussion by Randalix** [lima-vm/lima#998] [Q&A]
Can I use Lima to mount a btrfs disk on a m1 and create a samba share on that disk?
If I add /dev to the mounts I get:
```FATA[0020] the YAML is invalid, saved the buffer as "lima.REJECTED.yaml": field `mounts[2].location` must not be a system path such as /etc or /usr```
Sorry for the beginners question, but I've not found useful resources when searching about that topic.

**Comment by afbjorklund** [comment]
Mounting devices from the host is not really a beginner question, and would probably need support from Lima (and qemu) before it would work properly

Normally one would use a virtual disk image

* https://github.com/lima-vm/lima/issues/722

**Comment by msgilligan** [comment]
It would be nice to be able to use `limactl disk create` to create a Btrfs disk image and have it mounted via the `additionalDisks:` mechanism. The ability to use Btrfs snapshots would be really helpful for some use cases.

**Reply by AkihiroSuda** [reply]
Relevant:
- https://github.com/lima-vm/lima/pull/1516

**Reply by afbjorklund** [reply]
If I understand correctly, Fedora CoreOS uses btrfs for snapshots


EDIT: Close, it was Desktop: (but not our Cloud, nor CoreOS)
> The default file system on the desktop is Btrfs. It specifically excludes Fedora Server, **Cloud**, IoT, and CoreOS.

https://fedoraproject.org/wiki/Changes/BtrfsByDefault
